tests/qos/test_buffer.py

```diff
@@ -1681,9 +1769,11 @@ def _check_buffer_object_list_aligns_with_expected_ones(port_to_test, table, exp
 
     skip_traditional_model()
 
+    if not TESTPARAM_HEADROOM_OVERRIDE:
```

Comment:
If we update L226~239 according to my comments, we do not need it here. `TESTPARAM_HEADROOM_OVERRIDE` always exists

Reply:
Could you have a look at this comment? https://github.com/Azure/sonic-mgmt/pull/5784#discussion_r915055532

Reply:
The [comment](https://github.com/Azure/sonic-mgmt/pull/5784#discussion_r915055532) doesn't look correct.
However, my comment is not true as well. Even if it exists, it can be `None` and we still need check it because an exception will be thrown for `None.get()`
we can mark this comment as `resolved`